lib/membrane_rtmp_plugin/rtmp_server.ex

```diff
@@ -1,6 +1,14 @@
 defmodule Membrane.RTMP.Server do
   @moduledoc """
-  A simple RTMP server, which handles each new incoming connection.
+  A simple RTMP server, which handles each new incoming connection. When a new client connects, the new_client_callback is invoked.
+  New connections remain in an incomplete RTMP handshake state until another process makes demand for dara data.
+  If no data is demanded within the client_timeout period, the connection is closed.
```

Comment:
Perhaps it's good to emphasise that it's a client TCP socket that is closed under such circumstances?